iphone/Maps/Core/TextToSpeech/MWMTextToSpeech.h

```diff
@@ -15,9 +15,11 @@
 + (void)applicationDidBecomeActive;
 
 @property(nonatomic) BOOL active;
+@property(nonatomic) NSArray * testStrings;
```

Comment:
If this property shouldnt be modified from outside the class It is convenient to hide the setter in `.h`:
```objc
@interface MWMTextToSpeech : NSObject
@property(nonatomic, readonly) NSArray * testStrings;
@end
```
and open in `.m` by using the `readwrite`:
```objc
@interface MWMTextToSpeech ()
@property(nonatomic, readwrite) NSArray * testStrings;
@end
```
This will generate only the public getter. But inside the class getter+setter+ivar

OR

you can declare propety in `.m` and leave only the getter in `.h`:
`- (NSArray *) testStrings;`

Reply:
Alright I prefer the first version. Thanks!